c/driver/postgresql/connection.cc

```diff
@@ -23,18 +23,18 @@
 #include <adbc.h>
 
 #include "database.h"
-#include "util.h"
+#include "utils.h"
 
 namespace adbcpq {
 AdbcStatusCode PostgresConnection::Commit(struct AdbcError* error) {
   if (autocommit_) {
-    SetError(error, "Cannot commit when autocommit is enabled");
+    SetError(error, "%s", "[libpq] Cannot commit when autocommit is enabled");
```

Comment:
Instead of hard-coding libpq or SQLite another option would be to wrap this in a macro, use `__FILE__` and traverse up to the parent directory to get either `postgresql` or `sqlite`, etc...For now just stuck with hard coding libpq in this